### PR TITLE
Branch class created in Entites folder

### PR DESCRIPTION
**Pull request by AbbosJaan**
Branch class created in Entites folder. It is inherited from Auditable class.

### CODE REVIEW

**Comment by MuhammadKasimov** [comment]
Branch should be in Entities/Branches folder

File: BelissimoCloneWPF.Domain/Entities/Branchs/Branch.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿using BelissimoCloneWPF.Domain.Commons;
 
-namespace BelissimoCloneWPF.Domain.Entites.Branches
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Domain/Entities/Branchs/Branch.cs]: It should be Branches not Branchs
Create issue

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

**Comment by MuhammadKasimov** [comment]
Create issue

File: BelissimoCloneWPF.Domain/Entities/Branchs/Branch.cs (hunk 1 above)
Comment (AbbosJaan) [reply BelissimoCloneWPF.Domain/Entities/Branchs/Branch.cs]: I recently made changes and re-submitted them.

**Pull request merged**